src/device/device-management.spec.ts

```diff
@@ -20,6 +22,8 @@ describe('Device Management', () => {
   const mockStream = mocked(new MediaStream());
   const track = new MediaStreamTrack();
   mockStream.getTracks.mockReturnValue([track]);
+  mockStream.getVideoTracks.mockReturnValue([track]);
+  mockStream.getAudioTracks.mockReturnValue([track]);
```

Comment:
We have a helper to create a mocked stream in test-utils that does some of this...right now it's oriented around video streams but I think we could add another generic layer there that would just create a track and do all this stubbing out